### PR TITLE
Add submissions page

### PR DESCRIPTION
**Pull request by damianmarti**
localhost_3000_submissions

### CODE REVIEW

File: packages/nextjs/app/submissions/_components/SubmissionCard.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+"use client";
```
Comment (carletex) [inline packages/nextjs/app/submissions/_components/SubmissionCard.tsx]: We don't need it!

File: packages/nextjs/app/submissions/page.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { SubmissionCard } from "./_components/SubmissionCard";
+import type { NextPage } from "next";
+import { getAllSubmissions } from "~~/services/database/repositories/submissions";
+
+const Admin: NextPage = async () => {
```
Comment (carletex) [inline packages/nextjs/app/submissions/page.tsx]: Let's change the name to "Submissions"

File: packages/nextjs/app/submissions/page.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { SubmissionCard } from "./_components/SubmissionCard";
+import type { NextPage } from "next";
+import { getAllSubmissions } from "~~/services/database/repositories/submissions";
+
```
Comment (carletex) [inline packages/nextjs/app/submissions/page.tsx:5]: Let's add some metadata-

**Review by carletex — APPROVED** [review]
Thanks @damianmarti 

Posted a few comments, but just fixed it in d50a1ebe since it was small things

**Comment by damianmarti** [comment]
> Thanks @damianmarti
> 
> Posted a few comments, but just fixed it in [d50a1eb](https://github.com/BuidlGuidl/extensions-hackathon/commit/d50a1ebe3bed808fe4d2af7e68dc3a2126a52705) since it was small things

Thanks!!!

**Pull request merged**